
const funInfin = () => {
  while (true) {
    console.log("Nunca pares de Aprender")
  }
}


const fail = (message: string) => {
  throw new Error(message)
}